src/components/theme/View/ImageView.jsx

```diff
@@ -28,7 +28,7 @@ const ImageView = ({ content }) => (
       <p className="documentDescription">{content.description}</p>
     )}
     {content?.image?.download && (
-      <a href={flattenToAppURL(content.image.download)}>
+      <a href={addPrefixPath(flattenToAppURL(content.image.download))}>
```

Comment:
do not use addPrefixPath.. use UniversalLink instead <a> tag, it uses router that makes the magic with prefix path!

Reply:
Changed